obspy/taup/slowness_layer.py

```diff
@@ -110,11 +110,16 @@ def bullenDepthFor(layer, ray_param, radiusOfEarth):
             except OverflowError:
                 denom = np.inf
             A = np.divide(layer['topP'], denom)
-            with np.errstate(divide='ignore', invalid='ignore'):
+            if A != 0 and B != 0:
```

Comment:
Intuitively I'd make the if/else `if A == 0 or B == 0:` and put the below workaround in there, since these are the problematic cases.. but only cosmetics and personal taste I guess.